lib/uwb/include/uwb/protocols/fira/FiraDevice.hxx

```diff
@@ -75,6 +75,8 @@ constexpr uint8_t MaximumNumberOfStsSegmentsBprf = 1;
 constexpr uint8_t MaximumNumberOfStsSegmentsHprf = 4;
 constexpr uint8_t MinimumInBandTerminationAttemptCount = 0;
 constexpr uint8_t MaximumInBandTerminationAttemptCount = 10;
+constexpr uint8_t MinumumMandatoryMaximumNumberOfControleesInMulticastSession = 8;
```

Comment:
Typo: Should be `MinimumMandatoryMaximumNumberOfControleesInMulticastSession`

Reply:
@corbin-phipps When correcting typos, try to point out where exactly the typo is. I usually **bold** or _emphasize_ the part that's incorrect, eg.
Min**u**mumMandatoryMaximumNumberOfControleesInMulticastSession -> Min**i**mumMandatoryMaximumNumberOfControleesInMulticastSession 

But even better, you can use the suggestion feature in GitHub (Ctrl+g) or:
![image](https://user-images.githubusercontent.com/2082148/233732855-e9f9063b-51f1-4dfc-9013-b52ce33d78ae.png)

```suggestion
constexpr uint8_t MinimumMandatoryMaximumNumberOfControleesInMulticastSession = 8;
```